### PR TITLE
Add pressure names for SBE .cnv files

### LINKED ISSUE #80: Add SBE pressure names
Here is the list of pressure names for .cnv files:

- [ ] prM Pressure [db] pr M db User-entry for moored pressure (instrument with no pressure sensor)
- [ ] prE Pressure [psi] pr E psi User-entry for moored pressure (instrument with no pressure sensor)
- [x] prDM Pressure, Digiquartz [db] pr M db Digiquartz pressure sensor
- [ ] pr50M Pressure, SBE 50 [db] pr50 M db 1st SBE 50 pressure sensor
- [ ] pr50M1 Pressure, SBE 50, 2 [db] pr50 M2 db 2nd SBE 50 pressure sensor
- [ ] prSM Pressure, Strain Gauge [db] pr M db strain-gauge pressure sensor
- [x] prdM Pressure, Strain Gauge [db] pr M db strain-gauge pressure sensor
- [x] pr Pressure [db] Specific python-ctd name, see #20 

Marked items are recognized by cnv-parser.

### PR DESCRIPTION
**Pull request by vonopr**
Added support for following pressure names: prM, prE, pr50M, pr50M1, prSM.

Closes #80

### CODE REVIEW

**Comment by vonopr** [comment]
Travis-ci failed to build my merge. Log shows that problem is not related to the new commit. It is due to recent changes in matplotlib.

File: ctd/read.py [hunk 1]
```diff
@@ -380,8 +380,11 @@ def from_cnv(fname):
     )
     f.close()
 
+
     key_set = False
-    prkeys = ["prDM", "prdM", "pr"]
+    prkeys = [
+      "prM ", "prE", "prDM", "pr50M", "pr50M1", "prSM", "prdM", "pr"
```
Comment (ocefpaf) [inline ctd/read.py:386]: Can you add a comment on the top mentioned that these are the valid names for the seabird CNV format. If you have a link for that info please add that too.

File: ctd/read.py [hunk 2]
```diff
@@ -366,7 +366,7 @@ def from_cnv(fname):
 
     """
     f = _read_file(fname)
-    metadata = _parse_seabird(f.readlines(), ftype="cnv")
```
Comment (ocefpaf) [inline ctd/read.py:369]: Why `ftype="cnv"`? Where is that used?

Comment (ocefpaf) [reply ctd/read.py:369]: Argh. Sorry, that was my old change.

File: ctd/read.py [hunk 3]
```diff
@@ -351,7 +351,7 @@ def from_edf(fname):
     return df
 
 
-def from_cnv(fname):
+def from_cnv(fname, prname=None):
```
Comment (ocefpaf) [inline ctd/read.py:354]: Do we need the `prname`?

**Pull request merged**